__tests__/schema.test.js

```diff
@@ -45,12 +45,12 @@ test('path is return correct result and fillParametersFromArgs replace parameter
     {},
     {},
     { gender: 'female' }
-  );
-
-  expect(response).toMatchSnapshot();
+  ).then(response => {
+    expect(response).toMatchSnapshot();
```

Comment:
Why not async/await?

Reply:
Tests fail on node 6. If we want to add async/await then we need to add babel. 

Would you like to use babel?

Reply:
Meh, don't care. Anyway, you can always add a single plugin and use it just in test environment

Reply:
For now, I removed babel from the project for simplicity.

Reply:
cool, I'm good with that 👍